### ISSUE TITLE
403 error with portfolio function finviz.portfolio

### ISSUE THREAD

**Issue by parker-moore** [mariostoev/finviz#49]
Having the same issue with the Portfolio function:
using finviz 1.3.4

Traceback (most recent call last):
  File "c:\Users\parke\OneDrive\Desktop\StockTrading in Python\v.1.0\finviz porfolio.py", line 37, in <module>
    portfolio = Portfolio(USERNAME,PASSWORD)
  File "C:\Users\parke\AppData\Local\Programs\Python\Python38\lib\site-packages\finviz\portfolio.py", line 41, in __init__
    auth_response.raise_for_status()
  File "C:\Users\parke\AppData\Local\Programs\Python\Python38\lib\site-packages\requests\models.py", line 941, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 403 Client Error: Forbidden: Access is denied. for url: https://finviz.com/login_submit.ashx

_Originally posted by @parker-moore in https://github.com/mariostoev/finviz/issues/45#issuecomment-620249640_

**Comment by parker-moore** [comment]
@nguyendon I'm still having issues connecting. I updated my finviz files to match the new commits (which solved the 403 error) but when I try to create a new portfolio nothing happens. My code below:
portfolio = Portfolio('parkermoore4@outlook.com','*****')
for i in dict:
        temp = df
        for indicator in dict[i]:
            temp = eval(indicator)
        if temp.empty == True:
            continue
        temp = temp[['Ticker','Date','Close']]
        temp = temp.iloc[0:50] #grab first 50 stocks to invest in (finviz has a max of 50)
        temp['Transaction'] = 1 #1 means buy, 0 means sell
        temp = distribute_shares(48,temp)
        temp['Price'] = temp['Close']
        temp = temp[['Ticker','Transaction','Date','Shares','Price']]
        temp.set_index("Ticker",inplace=True)
        temp['Date'] = pd.to_datetime(temp['Date'])
        temp['Date'] = temp['Date'].dt.strftime("%d-%m-%Y")
        temp.to_csv(str(i) + '.csv')
        portfolio.create_portfolio(str(i),str(i) + '.csv')

**Comment by Will695** [comment]
I'm getting the same, even when i try just requests.get("http://finviz.com") i get a 403 error

**Comment by nguyendon** [comment]
@parker-moore I'd have to take a look, it seems to be another issue though. Can you file another issue? 

On the issue can you mention if you getting an error or if it just failing to create the portfolio?

**Comment by nguyendon** [comment]
@Will695 the issue is related to the user agent used by urllib being blocked by finviz. #50 addresses this. @mariostoev can you take a look at the PR?

**Comment by parker-moore** [comment]
It is not failing and there is no error. Runs smoothly just doesn't create portfolio on finviz @nguyendon
